### PR TITLE
Features

### PR DESCRIPTION
**Pull request by ferdelvalle**
I've been working on this project for a lot more time that I initially expected, this is the 5th time that I build it practically from scratch. Hence I was able to make it relatively fast. The three previous attempts were made using the source bootstrap files, and though the results were great, I never
managed to fully compile it into a single css file without getting compilation issues or the outcome being somehow broken. I got to learn a great deal
about SASS and I think overall it was a very humbling but teaching experience.

### CODE REVIEW

**Comment by pelzolga123** [comment]
## Hi Fernando!  
### Your project looks :+1: amazing! You did really good job.

* [ ] Tablet version has horizontal scrollbar, that should be removed:
image

* [ ] Mobile version has horizontal scrollbar too, and `More stories` section is a little messed up:
image

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by ferdelvalle** [comment]
As required, horizontal bars fixed 
read-me
and
read-me2

File: index.html [hunk 1]
```diff
@@ -0,0 +1,926 @@
...
+          <div class="right-arrow grey"></div>
+        </a>
+        <a href="#closed"
+          class="bottom-nav-anchor-hidden pl-3 py-1 d-flex justify-content-between align-items-end"><span>Close
+            Menu</span>
+          <div class="genericon genericon-close-alt grey exit"></div>
+        </a>
+      </nav>   
+    </div>
+  </header class="width-fix">
```
Comment (MauricioRobayo) [inline index.html]: This is a closing tag! No attributes here 😄 .

Comment (ferdelvalle) [reply index.html]: My bad! but already corrected, thank you so much

**Pull request merged**